README.md

```diff
@@ -68,86 +89,55 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
   job that creates realistic usage patterns on the website using
   [Locust](https://locust.io/) load generator.
 
-## Installation
+## Installation Options
+You can install the Hipster-Shop into these environments:
+* [**Google Kubernetes Engine** (GKE)](#gke-installation): Install the app to a similar environment that you'll deploy your production system to.
+* [**Locally**](#local-installation): Deploy to a single-node Kubernetes cluster running on your own development machine, using either [Minikube](https://github.com/kubernetes/minikube) (recommended for Linux) or [Docker for Desktop](https://www.docker.com/products/docker-desktop) (recommended for Mac/Windows).
+Both installations take between 20-30 minutes
+
+## Prerequisites
+### GKE Installation Only
+* **Google Cloud Platform account**: Visit [console.cloud.google.com](https://console.cloud.google.com) to set up your account.<br>
+Create and activate your project.
+
+* **Set the project ID as an environment variable**<br>
+    `export GCP_PROJECT_ID=<your-project-name>`
+
+* **Google Cloud Console (command line tool)**: Visit [cloud.google.com/pubsub/docs/quickstart-cli](https://cloud.google.com/pubsub/docs/quickstart-cli) for instructions.
 
-We offer the following installation methods:
+* Run `gcloud init` to configure Google Cloud (GC). Follow the console instructions. If you've already created a project in GC, be sure to select that project.
+  > 💡 Make sure to include the whole name of the project, including any dashes or numbers after the name.
```

Comment:
That's actually the difference between the ID and the name. The ID is the long string.